source/export-to-language.txt

```diff
@@ -0,0 +1,44 @@
+.. _vsce-export-to-language:
+
+======================================
+Export a Query or Pipeline to Language
+======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can export query documents and aggregation pipelines from a
```

Comment:
Are query documents and aggregation pipelines the only things you can export? (The shell will take variables as well. Not sure what the drives can bring in.)

Maybe something more general like,

"You can export code developed in  :ref:`playground <vsce-playgrounds>` " ...

Reply:
You can only export query documents and aggregation pipelines.

Reply:
Done